plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx

```diff
@@ -19,7 +19,6 @@ import {
   ContentHeader,
```

Comment:
There would be no changes in this file for this PR, it's just an example to try it out.